### PR TITLE
Implement quantizeLinear

### PR DESCRIPTION
**Pull request by BruceDai**
@huningxin @fdwr @lisa0314 PTAL, thanks.

### CODE REVIEW

File: src/quantize_linear.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * The calculation follows the expression roundToNearestEvens(input / scale) + zeroPoint.
+ * @param {Tensor} input
+ * @param {Tensor} scale
+ * @param {Tensor} zeroPoint
+ * @param {String} dataType
+ * @return {Tensor}
+ */
+export function quantizeLinear(input, scale, zeroPoint, dataType) {
+  const addOutput = add(div(input, scale), zeroPoint);
+  const roundOutput = unary(addOutput, (x) => roundToNearestEvens(x));
```
Comment (fdwr) [inline src/quantize_linear.js:23]:
```suggestion
  const dividedOutput = div(input, scale);
  const roundedOutput = unary(dividedOutput, (x) => roundToNearestEvens(x));
  const addedOutput = add(roundedOutput, zeroPoint);
```
Round before adding.

File: test/quantize_linear_test.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    const inputTensor = new Tensor(input.shape, input.value);
+    const scaleTensor = new Tensor(scale.shape, scale.value);
+    const zeroPointTensor = new Tensor(zeroPoint.shape, zeroPoint.value);
+    const outputTensor = quantizeLinear(inputTensor, scaleTensor, zeroPointTensor, dataType);
+    utils.checkShape(outputTensor, expected.shape);
+    utils.checkValue(outputTensor, expected.value);
+  }
+
+  it('quantizeLinear 1D to uint8 broadcasting scale and zeroPoint', function() {
+    testQuantizeLinear(
```
Comment (fdwr) [inline test/quantize_linear_test.js:18]: A 0D case would complete the set.

**Review by fdwr — COMMENTED** [review]
Small comments, else LGTM. 🫡

File: test/quantize_linear_test.js (hunk 2 above)
Comment (BruceDai) [reply test/quantize_linear_test.js:40]: Done, thanks.

File: src/quantize_linear.js (hunk 1 above)
Comment (BruceDai) [reply src/quantize_linear.js]: Updated, thanks.

**Review by BruceDai — COMMENTED** [review]
@fdwr I've submitted new commit, please take another look, thanks.

**Review by fdwr — APPROVED** [review]
👍

**Review by huningxin — APPROVED** [review]

**Pull request merged**